status/flowcells.py

```diff
@@ -344,17 +344,18 @@ def get(self, query):
             xfc_view = self.application.x_flowcells_db.view("samples/lane_clusters", reduce=False)
             fc_view = self.application.flowcells_db.view("samples/lane_clusters", reduce=False)
             for row in xfc_view[query:"{}Z".format(query)]:
-                if not "isHiseqX" in data:
+                if not "isHiseqX" in data: # also don't need
                     data["isHiseqX"]=True
                 if not row.key in data:
                     data[row.key]=[]
                 data[row.key].append(row.value)
-            for row in fc_view[query:"{}Z".format(query)]:
-                if not row.key in data:
-                    data[row.key]=[]
-                if row.value not in data[row.key]:
-                    data[row.key].append(row.value)
-            for key in sorted(data.keys()):
+            # approved by FV, removing these lines 'MOST LIKELY it won't fail'
+            # for row in fc_view[query:"{}Z".format(query)]:
+            #     if not row.key in data:
+            #         data[row.key]=[]
+            #     if row.value not in data[row.key]:
+            #         data[row.key].append(row.value)
+            for key in sorted(data.keys()): # and these lines we also don't need
                 if key == "isHiseqX":
```

Comment:
For now let us merge like this  but would be better if @kate-v-stepanova can give a closer look to this and remove completely the "isHiSeqX" as it should not be needed at all